src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -923,6 +923,26 @@ constexpr static const auto EXPORT_IMPORT = "EXPORT_IMPORT";  //!< Device suppor
 /** @}*/
 }  // namespace capability
 
+/**
+ * @brief Read-write property to set alias for hardware plugin
+ * value type: string Alias name for the set of plugins
+ * @ingroup ov_runtime_cpp_prop_api
+ */
+static constexpr Property<std::string, PropertyMutability::RW> alias{"DEVICE_ALIAS"};
```

Comment:
May it be better to move out from common properties to internal `proxy` properties?

Reply:
These properties are related only for real plugins